tests/static_multimap/static_multimap_test.cu

```diff
@@ -83,7 +83,7 @@ __inline__ void test_custom_key_value_type(Map& map,
                                            KeyIt key_begin,
                                            size_t num_pairs)
 {
-  constexpr cudaStream_t stream = 0;
+  constexpr cudaStream_t stream{nullptr};
```

Comment:
I am curious about if we can use `nullptr` instead of `0`.
https://docs.nvidia.com/cuda/cuda-c-programming-guide/index.html#default-stream
> Kernel launches and host <-> device memory copies that do not specify any stream parameter, or equivalently that set the stream parameter to zero, are issued to the default stream. They are therefore executed in order.

Reply:
Right, `0` is the common way to initialize CUDA stream while according to the CUDA user guide, `cudaStream_t` is a type alias of `CUstream_st *` (see [here](https://docs.nvidia.com/cuda/cuda-runtime-api/group__CUDART__TYPES.html#group__CUDART__TYPES_1ge15d9c8b7a240312b533d6122558085a)) and `clang-tidy` will complain if it's initialized with `0`. @karthikeyann @sleeepyjack For your reference, `rmm` is doing the same in https://github.com/rapidsai/rmm/pull/857.